src/main/java/screen/GameScreen.java

```diff
@@ -288,8 +293,11 @@ public final void initialize() {
 		this.screenFinishedCooldown = Core.getCooldown(SCREEN_CHANGE_INTERVAL);
 		this.bullets = new HashSet<>();
 		this.barriers = new HashSet<>();
+		this.allyShips = new HashSet<>();
         this.itemBoxes = new HashSet<>();
-		this.itemManager = new ItemManager(this.ship, this.enemyShipFormation, this.barriers, this.width, this.height, this.balance);
+		this.itemManager = new ItemManager(this.ship, this.enemyShipFormation,
+				this.barriers, this.allyShips,
+				this.height, this.width, this.balance);
```

Comment:
The order of parameters in this code needs to be changed. It seems there was some confusion during the rebase process.
```suggestion
		this.itemManager = new ItemManager(this.ship, this.enemyShipFormation, this.barriers, this.allyShips, this.width, this.height, this.balance);
```